docs/snippets/vue/list-story-with-sub-components.vue3.js.mdx

```diff
@@ -0,0 +1,31 @@
+```js
+// List.stories.js
+
+import List from './List.vue';
+import ListItem from './ListItem.vue';
+
+export default {
+  component: List,
+  subcomponents: { ListItem }, //👈 Adds the ListItem component as a subcomponent
+  title: 'List',
+};
+
+export const Empty = (args) => ({
+  props: Object.keys(argTypes),
```

Comment:
```suggestion
```

No longer needed for vue3! 🎉

Reply:
@phated thank you for catching that one 👍